sympy/physics/units/definitions/unit_definitions.py

```diff
@@ -114,6 +108,21 @@
 becquerel = Bq = Quantity("becquerel", abbrev="Bq")
 
 
+# Common mass units
+
+mg = milligram = milligrams = Quantity("milligram", abbrev="mg")
+mg.set_global_relative_scale_factor(milli, gram)
+
+ug = microgram = micrograms = Quantity("microgram", abbrev="ug", latex_repr=r"\mu\text{g}")
+ug.set_global_relative_scale_factor(micro, gram)
+
+# Atomic mass constant
+Da = dalton = amu = amus = atomic_mass_unit = atomic_mass_constant = Quantity("atomic_mass_constant")
+
+t = tonne = Quantity("tonne", abbrev="t")
+t.set_global_relative_scale_factor(kilo, kg)
```

Comment:
Weight are a bit confusing since the SI unit is kg and not gram, so not really clear if tonne should be Mg or kkg... Should there even be one? Compare with hectare which doesn't have one. (hundred are, one are = 100 m^2).

Reply:
> Weight are a bit confusing since the SI unit is kg and not gram, so not really clear if tonne should be Mg or kkg... Should there even be one? Compare with hectare which doesn't have one. (hundred are, one are = 100 m^2).

Firstly I think you are talking about mass because kg and gram are it's units. You can checkout `unit_definitions.py` , where `set_global_relative_scale_factor ` has been without much attention to whether it's with the SI unit of the quantity or not ,like yard is set with respect to feet . If you are concerned with unit conversions then we always have` convert_to ` function and coincidently I have converted tonne to Mg in the test added .So I don't think we should have any issues here.
Secondly when I was making the changes in this file I saw that only units representing fundamental quantities had were subject to `set_global_relative_scale_factor` which makes sense . Hectare is a unit of area hence doesn't have one similar to why Litre for volume doesn't have one similar to why Newton for force doesn't have one .Even here I believe the current diff is appropriate.

Reply:
CC.@dyc3 This is the only unresolved issue in this PR . Since you are a contributor to `physics.units` I would like to have your thoughts on relevant comments !

Reply:
From what I can see, I think it would make more sense to have it be:

```suggestion
tonne.set_global_relative_scale_factor(mega, gram)
```

(nit) We should probably alias `tonne` to `metric_ton` as well.
(nit) I think `t` for `tonne` is confusing. I've never seen tonne referred to as t out in the wild.
(nit) I think you should prefer `kilogram` over `kg`.

Reply:
1. Adding `metric_ton` makes sense ,I will do that soon.
2. I looked into online literature for `t` .It is a valid symbol for `tonne `,checkout [wikipedia](https://en.wikipedia.org/wiki/Non-SI_units_mentioned_in_the_SI) .  So maybe I can make that change here but not remove all t instances from the diff.
3. Not sure where you would want me to prefer `kilogram` over `kg`. Since you have suggested to change `kg` into `gram` above.

Reply:
I meant prefer the longer name for a unit instead of the shorter abbreviated name. (so prefer `gram` over `g`) In general, I think it makes it easier to read. The only difference is cosmetic, hence the (nit).

Reply:
Thanks .I get your point .

Reply:
I have made a commit now ,I hope that the changes are appropriate !

Reply:
I have made the changes and all checks have passed .Unless there are some particular doubts related to this I intend to resolve this issue soon . Thanks for the reviews !